view/templates/email/html.tpl

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional //EN">
+<html>
+<head>
+	<title>{{$banner}}</title>
+	<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+</head>
+<body>
+	<table style="border:1px solid #ccc">
+	<tbody>
```

Comment:
Do we need a table here? Wont be a list and a div with some styling look alike? I don't see "row" and "cell" oriented structure in the template.

Reply:
Email styling is tricky, clients vary wildly, and some purely ignore provided styles. Tables are the only way to ensure layout is respected across clients.

Reply:
Well, in my personal stats, HTML mail is a reasonable well indicator for the mail being SPAM anyway ;-) but if a table is _the only way_ to get the styling through to the client, then a table it has to be.

Reply:
So I don't have to redesign the whole email layout _*phew*_